superset/assets/src/explore/components/controls/SubscriberLayer.jsx

```diff
@@ -64,8 +58,6 @@ const defaultProps = {
   allowMoreColumns: false,
   allowColumnSelection: false,
```

Comment:
allowColumnSelection use?

Reply:
allowColumnSelection allows column selection if any slice selected otherwise not